### PR TITLE
Docs: Updates readme file - FastStore v2

### PR DESCRIPTION
**Pull request by hellofanny**
## What's the purpose of this pull request?

- Updates readme for FastStore v2

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,33 +18,20 @@
 - <strong>Performance</strong>: Keep yourself calm. We use strict budgets to ensure our libs are always light and fast.
 - <strong>Stability</strong>: Generate your pages ahead by using a Jamstack ready toolkit.
 - <strong>SEO/Analytics</strong>: Connect to your favorite analytics provider.
-- <strong>Freedom</strong>: Use our API/SDK/UI components to connect, manage and display your ecommerce in your favorite React framework
-  (Gatsby, Next.JS, Remix etc)
-- <strong>Speed</strong>: Customize one of our starters to your needs. Do not waste time creating tests, thinking about architecture or infrastructure, it's all taken care of.
+- <strong>Freedom</strong>: Use our API/SDK/UI components to connect, manage and display your ecommerce using Next.JS, a powerful React framework
```
Comment (hellofanny) [inline README.md:21]:
```suggestion
- <strong>Freedom</strong>: Use our API/SDK/UI components to connect, manage and display your ecommerce using Next.JS, a powerful React framework.
```

File: README.md [hunk 2]
```diff
@@ -56,11 +43,11 @@ If you find any issues on the project you would like to report, please create an
...
+| @faststore/api | Connect your site to your ecommerce provider                                                                       |                                                                                                                                                                                             [![npm version](https://badge.fury.io/js/%40faststore%2Fapi.svg)](https://bundlephobia.com/package/@faststore/api)                                                                                                                                                                                              |
```
Comment (hellofanny) [inline README.md:50]: Maybe we should add `faststore/core` here too.

File: README.md [hunk 3]
```diff
@@ -18,33 +18,20 @@
...
 
-[Check out the docs](https://faststore.dev/quickstart) to quickstart from one of our official starters or create your own.
+[Check out the docs](https://faststore.dev/) to quickstart from our official starter and start creating.
 
-## Official Starters
+## Official Starter
 
-- [Gatsby](https://github.com/vtex-sites/gatsby.store): Basic starter with a generic store.
-- [NextJS](https://github.com/vtex-sites/nextjs.store): Basic starter with a generic store.
+- [NextJS](https://github.com/vtex/faststore): Basic starter with a generic store.
```
Comment (hellofanny) [inline README.md:30]:
```suggestion
- [NextJS](https://github.com/vtex-sites/starter.store):  FastStore Starter for a generic store.
```

[the remark this answers is not in the document]
Comment (Mariana-Caetano) [reply README.md:30]: I think makes sense to link to the starter repo.

File: README.md (hunk 2 above)
Comment (Mariana-Caetano) [reply README.md:50]: Agreed :)

File: README.md [hunk 4]
```diff
@@ -18,33 +18,20 @@
...
-- <strong>Freedom</strong>: Use our API/SDK/UI components to connect, manage and display your ecommerce in your favorite React framework
-  (Gatsby, Next.JS, Remix etc)
-- <strong>Speed</strong>: Customize one of our starters to your needs. Do not waste time creating tests, thinking about architecture or infrastructure, it's all taken care of.
+- <strong>Freedom</strong>: Use our API/SDK/UI components to connect, manage and display your ecommerce using Next.JS, a powerful React framework.
+- <strong>Speed</strong>: Customize our starter to fit your needs. Do not waste time creating tests, thinking about architecture or infrastructure, it's all taken care of.
 
 ## Getting started
 
-[Check out the docs](https://faststore.dev/quickstart) to quickstart from one of our official starters or create your own.
+[Check out the docs](https://faststore.dev/) to quickstart from our official starter and start creating.
```
Comment (Mariana-Caetano) [inline README.md:26]:
```suggestion
[Check out the docs](https://www.faststore.dev/docs/getting-started-overview) to quickstart from our official starter and start creating.
```

File: README.md [hunk 5]
```diff
@@ -18,33 +18,20 @@
...
-
-## Who's using FastStore?
-
-These are the clients running in production with FastStore:
-
-- [Marin Brasil](https://www.marinbrasil.com.br/)
-- [Carrefour](https://mercado.carrefour.com.br/)
+Check out our [Overview](https://www.faststore.dev/docs/getting-started-overview), [How-to Guides](https://www.faststore.dev/docs/building-sections/component-customization-overview), [API References](https://faststore-site-git-api-extension-development-faststore.vercel.app/docs/api-extensions) and [Glossary](https://www.faststore.dev/docs/getting-started-glossary) to deep dive in the world of building FastStores.
 
 ## Contributing
```
Comment (Mariana-Caetano) [inline README.md:36]: What about a link to the [CONTRIBUTING.MD](https://github.com/vtex/faststore/blob/docs/update-readme/CONTRIBUTING.MD) as well? For example:

> This is still a work in progress. However, if you are also adventurous, you can read the code and have fun! Remember to check out our  [CONTRIBUTING.MD](https://github.com/vtex/faststore/blob/docs/update-readme/CONTRIBUTING.MD).

**Review by Mariana-Caetano — COMMENTED** [review]
Awesome, Fanny! ✨
Left some suggestions and comments.

File: README.md (hunk 5 above)
Comment (hellofanny) [reply README.md:36]: sounds good, but I think this file is out of date :/

File: README.md [hunk 6]
```diff
@@ -56,11 +43,11 @@ If you find any issues on the project you would like to report, please create an
...
 | @faststore/ui  | Exports basic Store components. Check out our [documentation](https://www.faststore.dev/reference/ui/faststore-ui) |   [![npm version](https://badge.fury.io/js/%40faststore%2Fui.svg)](https://badge.fury.io/js/%40faststore%2Fui) [![size minzip](https://badgen.net/bundlephobia/minzip/@faststore/ui)](https://bundlephobia.com/package/@faststore/ui) [![tree shaking](https://badgen.net/bundlephobia/tree-shaking/@faststore/ui)](https://bundlephobia.com/package/@faststore/ui) [![dependency count](https://badgen.net/bundlephobia/dependency-count/@faststore/ui)](https://bundlephobia.com/package/@faststore/ui)   |
```
Comment (hellofanny) [inline README.md:48]:
```suggestion
| @faststore/core  | Bundles FastStore source code, including the starter for FatsStore projects: starter store. It contains three sub-packages, Components, SDK, and API, which handle the starter.) |   [![npm version](https://badge.fury.io/js/%40faststore%2Fcore.svg)](https://badge.fury.io/js/%40faststore%2Fcore) [![size minzip](https://badgen.net/bundlephobia/minzip/@faststore/core)](https://bundlephobia.com/package/@faststore/core) [![tree shaking](https://badgen.net/bundlephobia/tree-shaking/@faststore/core)](https://bundlephobia.com/package/@faststore/core) [![dependency count](https://badgen.net/bundlephobia/dependency-count/@faststore/core)](https://bundlephobia.com/package/@faststore/core)   |
| @faststore/ui  | Exports basic Store components. Check out our [documentation](https://www.faststore.dev/reference/ui/faststore-ui) |   [![npm version](https://badge.fury.io/js/%40faststore%2Fui.svg)](https://badge.fury.io/js/%40faststore%2Fui) [![size minzip](https://badgen.net/bundlephobia/minzip/@faststore/ui)](https://bundlephobia.com/package/@faststore/ui) [![tree shaking](https://badgen.net/bundlephobia/tree-shaking/@faststore/ui)](https://bundlephobia.com/package/@faststore/ui) [![dependency count](https://badgen.net/bundlephobia/dependency-count/@faststore/ui)](https://bundlephobia.com/package/@faststore/ui)   |
```

File: README.md [hunk 7]
```diff
@@ -56,11 +43,12 @@ If you find any issues on the project you would like to report, please create an
...
```
Comment (hellofanny) [inline README.md:49]:
```suggestion
| @faststore/ui  | Exports basic Store components. Check out our [documentation](https://www.faststore.dev/components) |   [![npm version](https://badge.fury.io/js/%40faststore%2Fui.svg)](https://badge.fury.io/js/%40faststore%2Fui) [![size minzip](https://badgen.net/bundlephobia/minzip/@faststore/ui)](https://bundlephobia.com/package/@faststore/ui) [![tree shaking](https://badgen.net/bundlephobia/tree-shaking/@faststore/ui)](https://bundlephobia.com/package/@faststore/ui) [![dependency count](https://badgen.net/bundlephobia/dependency-count/@faststore/ui)](https://bundlephobia.com/package/@faststore/ui)   |
```

**Review by Mariana-Caetano — APPROVED** [review]

**Pull request merged**